frontend/express/public/core/notes/javascripts/countly.views.js

```diff
@@ -68,12 +82,22 @@
                 case 'edit':
                     data.color = {value: data.color};
                     this.drawerSettings.isEditMode = true;
+                    data.note = this.decodeHtml(data.note);
```

Comment:
Can you call this here:
https://github.com/Countly/countly-server/blob/master/frontend/express/public/javascripts/countly/countly.common.js#L294
+if something missing - make additions in that common function.